public/react/SaveReports/SaveReports.tsx

```diff
@@ -116,16 +116,14 @@ const SaveReport: FC<SaveReportProps> = ({ queryParamObservable, onViewReportsBu
             )}
           >
             <DropdownItem
-              onClick={(event) => {
-                event.stopPropagation();
+              onClick={() => {
                 updateSelectedReport();
               }}
             >
               Save changes
             </DropdownItem>
             <DropdownItem
-              onClick={(event) => {
-                event.stopPropagation();
```

Comment:
this was apparently keeping focus on the menu item and wasn't allowing the text to be focused

Reply:
Does it work with the routing?